scripts/tests/test_validate_docstrings.py

```diff
@@ -146,6 +146,37 @@ def head1(self, n=5):
         """
         return self.iloc[:n]
 
+    def order(self):
+        """
+        In this case, we are illustrating the correct ordering
```

Comment:
If this is a example of good docstring , this summary should finish with a period and fit in one line. Tests should be failing because of it if this docstring is tested.